x-pack/platform/packages/shared/index-management/index_management_shared_types/src/types.ts

```diff
@@ -35,9 +36,14 @@ export type IndexManagementLocatorParams = SerializableRecord &
 
 export type IndexManagementLocator = LocatorPublic<IndexManagementLocatorParams>;
 
+export type IndexManagementAppMountParams = Pick<
```

Comment:
Perfect! I was really confused previously about how in one situation one set of params would work and another one would need a different set of params.